enterprise/api_client/sso_orchestrator.py

```diff
@@ -131,4 +132,5 @@ def configure_sso_orchestration_record(
         if is_sap or sap_config_data:
             request_data['sapsfConfiguration'] = sap_config_data
 
-        return self._post(self._get_orchestrator_configure_url(), data=request_data)
+        response = json.loads(self._post(self._get_orchestrator_configure_url(), data=request_data).text)
```

Comment:
with the other method returning `response.json()` you wont need a json parse here, which might make this section of code clearer?